main.py

```diff
@@ -21,7 +21,8 @@ def __init__(self):
         self.customers: List[Customer] = []
         self.active_sessions: dict[str, Customer] = {}
 
-    def create_account(self, account_type: str = "Normal", initial_balance: float = 0) -> Customer:
+    @staticmethod
+    def create_account(account_type: str = "Normal", initial_balance: float = 0) -> Account:
```

Comment:
Should be an account not associated with a bank on creation? Can you move bank accounts between different banks?